### PR TITLE
sample: Add name "sample" in the some samples modules

### PR DESCRIPTION
**Pull request by katgiadla**
Some test cases in the module `samples` have not a name of this module.
This change adds the name `sample ` to these test cases.

Signed-off-by: Katarzyna Giadla <katarzyna.giadla@nordicsemi.no>

### CODE REVIEW

**Review by kkasperczyk-no — APPROVED** [review]

**Review by sebastiandraus — APPROVED** [review]

**Comment by kkasperczyk-no** [comment]
@katgiadla please consider changes done in https://github.com/nrfconnect/sdk-nrf/pull/7271 for the matter template sample.yaml: https://github.com/nrfconnect/sdk-nrf/pull/7271/files#diff-d49928f22e2d93edf4184858d9bfc98ebed7820455930bfd8ec1f563e20fcd67R22

File: samples/matter/template/sample.yaml [hunk 1]
```diff
@@ -1,29 +1,21 @@
...
-  matter.template.release:
-    build_only: true
     platform_allow: nrf52840dk_nrf52840 nrf5340dk_nrf5340_cpuapp nrf21540dk_nrf52840
-    integration_platforms:
-      - nrf52840dk_nrf52840
-      - nrf5340dk_nrf5340_cpuapp
-      - nrf21540dk_nrf52840
     tags: ci_build
-    extra_args: CONF_FILE=prj_release.conf
-  matter.template.no_dfu:
```
Comment (kkasperczyk-no) [inline samples/matter/template/sample.yaml:22]: Please rename step for sample.matter.template.no_dfu: and do not remove it.

**Review by kkasperczyk-no — CHANGES_REQUESTED** [review]

**Review by kkasperczyk-no — APPROVED** [review]

File: samples/matter/template/sample.yaml (hunk 1 above)
Comment (katgiadla) [reply samples/matter/template/sample.yaml:22]: Done

**Review by Damian-Nordic — APPROVED** [review]

**Review by tomchy — APPROVED** [review]

**Review by totyz — APPROVED** [review]
CI adapted to new sample names. Please re-run CI for Zigbee

**Pull request merged**